src/test/scala/com/datastax/gatling/plugin/base/BaseCassandraServerSpec.scala

```diff
@@ -45,12 +46,11 @@ class BaseCassandraServerSpec extends BaseSpec {
 }
 
 object BaseCassandraServerSpec {
-  if (EmbeddedCassandraServerHelper.getSession == null) {
```

Comment:
The original cause of all failures for cassandra-unit based tests after upgrading to 4.2.2.0-SNAPSHOT.  In the new version getSession() calls initSession() to make sure it's got a good session to return... and in so doing it has to access several methods on DatabaseDescriptor.  But in our usage we haven't even started the embedded C* instance yet, and it's that act that is responsible for setting up cassandra.yaml in a known spot (and thus loading DatabaseDescriptor).  So our tests were always going to fail.

We're in an object here so singleton behaviour is already guaranteed to us... so the check was largely superfluous.